rules/framework.bzl

```diff
@@ -307,7 +306,11 @@ def _apple_framework_packaging_impl(ctx):
     # Eventually we need to remove any reference to objc provider
     # and use CcInfo instead, see this issue for more details: https://github.com/bazelbuild/bazel/issues/10674
     objc_provider = apple_common.new_objc_provider(**objc_provider_fields)
-    cc_info_provider = CcInfo(compilation_context = objc_provider.compilation_context)
+    cc_info_provider = CcInfo(
```

Comment:
do we need to do something to include the transitive compilation contexts / ccinfos ?

Reply:
@segiddins I _think_ the `merge_cc_infos` below does this?

Reply:
🤦🏻  indeed it does. my bad for not reading more carefully